### PR TITLE
Implemente book view.

### PR DESCRIPTION
**Pull request by mpizzagalli**
# Summary
* Added a link to the name of each book in the dashboard, that redirects us to a bigger view of the book details.

# Trello Card
https://trello.com/c/ovuU9yMQ/7-crear-vista-de-detalle-de-un-libro

# Screenshots
dash_links
book_view

### CODE REVIEW

File: src/app/components/books/booksController.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+angular.module('app').controller('BooksController', [
+  '$state', '$stateParams', 'booksService',
+  function ($state, $stateParams, booksService) {
+    this.id = $stateParams.id;
```
Comment (sbalay) [inline src/app/components/books/booksController.js]: why do you need this?
you can just 
`booksService.getBook($stateParams.id)...`

File: src/app/components/books/booksController.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+    const success = (response) => {
```
Comment (sbalay) [inline src/app/components/books/booksController.js]: we usually do not declare a new variable for callbacks. just
```
booksService.getBook($stateParams.id).then((response) => {
  this.book = response;
});
```

Comment (mpizzagalli) [inline src/app/components/books/booksController.js]: I saw that it was done like that in Increase so thought of doing it also here. Maybe it has more sense for callbacks that are a little more complex compared to this case, what do you think?

Comment (sbalay) [inline src/app/components/books/booksController.js]: In my opinion, declaring the callback as a variable in the controller makes sense if we are going to reuse that callback function for any other action in the controller.
Don't need to change this, looks good.

**Comment by sbalay** [comment]
:+1:

**Pull request merged**